### ISSUE TITLE
NTR: pre-coordinated terms for accumulation in PATO

### ISSUE THREAD

**Issue by Achchuthan** [pato-ontology/pato#182]
Hi,

I am looking at representing some chemical and RNA accumulation phenotypes in EuPathDB. For the quality terms, I find "accumulation", "increased accumulation" and "decreased accumulation" in PATO. Can I request three more pre-coordinated terms to represent "normal accumulation", "abnormal accumulation" and "arrested/ abolished/ absence of accumulation" please?

Thank you very much for your help.

Best wishes,
Achchuthan

**Comment by drseb** [comment]
Normal and abnormal states are usually represented as modifiers. See e.g. the logical definition here: http://www.ontobee.org/ontology/MP?iri=http://purl.obolibrary.org/obo/MP_0005059

I do not think there will be pre-composed terms for normal/abnormal/etc. in PATO. 

ping @cmungall

**Comment by Achchuthan** [comment]
Hi @drseb,
Thanks a lot for your quick response and the suggestion. 

There are normal states represented in PATO such as "normal resistance to (PATO:0045047)" and "normal sensitivity toward (PATO:0045049). I do find several others as well. I also wanted to reference an earlier issue (#139 ), where it has been discussed about adding post-coordinated terms on a need basis.

In EuPathDB, we use EQ pattern to annotate and query phenotype data. Currently, we have two columns for phenotypes, one column for E and one for Q. Each column only allows one ontology term. So, for qualifier, it is better to have some pre-coordinated PATO, as discussed by Jie Zheng in issue #139 .

Thanks & best wishes,
Achchuthan

**Comment by drseb** [comment]
I agree that PATO is not really consistent, given the example that you provided. I think others might want to comment on that issue (I hope), but just one last comment from my side. Have you had a look at how ZFIN represents their data? (see Zfin.org)

<img width="1672" alt="screen shot 2018-09-21 at 13 46 25" src="https://user-images.githubusercontent.com/5172897/45879541-33233900-bda5-11e8-9ba3-48b69ce5140a.png">

Maybe that gives you an idea in another direction than adding precomposed terms to PATO, which is of course also a valid path (esp. given the referenced issue/ticket). I think @cmungall @gkoutos or any other more experienced PATO-editor can take it from here.
